common/djangoapps/student/models.py

```diff
@@ -1506,10 +1506,13 @@ def change_mode(self, mode):
         """Changes this `CourseEnrollment` record's mode to `mode`.  Saves immediately."""
         self.update_enrollment(mode=mode)
 
-    def refundable(self):
+    def refundable(self, user_already_has_certs_for=None):
         """
         For paid/verified certificates, students may receive a refund if they have
         a verified certificate and the deadline for refunds has not yet passed.
```

Comment:
Er... it would seem to directly contradict assumptions elsewhere in the code. I'll investigate. Who's the expert on that?